SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2590,39 +2803,78 @@ void USpatialNetDriver::TryFinishStartup()
 
 	if (IsServer())
 	{
-		if (!PackageMap->IsEntityPoolReady())
```

Comment:
This diff boils down to putting the regular server startup sequence behind
`if (WorkerType == SpatialConstants::DefaultServerWorkerType)`